### PR TITLE
Provide install script

### LINKED ISSUE #140: Bash script with all the commands
A bash script containing all the commands from the tutorial to make it easier to install the node

### PR DESCRIPTION
**Pull request by happyhackerbird**
Fixes #140
(HERDAO collaboration)

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+# Clone simple-taiko-node
+# git clone https://github.com/taikoxyz/simple-taiko-node.git
+# cd simple-taiko-node
+# printf "\n"
+
+echo Taiko Node Setup Script v0
+
+# Copy the sample .env files
+echo Getting environmental variables...
```
Comment (d1onys1us) [inline install.sh:11]:
```suggestion
echo Getting environment variables...
```

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+cp .env.sample .env
+
+# get api key
+echo Setting L1 archive node endpoints
```
Comment (d1onys1us) [inline install.sh:15]:
```suggestion
echo Setting L1 archive node endpoints...
```

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+sudo docker compose up -d > /dev/null
+printf "\xE2\x9C\x94 Node started! \n"
+printf "\n"
+
+# Verify node is running
+echo Verifying node is running...
+
+# Check if the Execution Layer client is connected to Taiko L2
+echo Check the Execution Layer client is connected to Taiko L2
+curl -s http://localhost:8547 \
```
Comment (d1onys1us) [inline install.sh:38]: i would recommend reading these port values from the .env.sample file in case they change. same with line 68

**Review by d1onys1us — COMMENTED** [review]
two nitpicks, otherwise looks good to me. thanks for your contribution!

**Comment by davidtaikocha** [comment]
Closed due to inactive.

**Pull request closed without merging**